OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1686,7 +1686,7 @@ void Model::writeMarkerFile(const string& aFileName)
  * @param aMarkerSet The new marker set to copy.
  * @return Number of markers that were successfully added to the model.
  */
-int Model::replaceMarkerSet(const SimTK::State& s, const MarkerSet& aMarkerSet)
+int Model::replaceMarkerSet(SimTK::State& s, const MarkerSet& aMarkerSet)
```

Comment:
I also noticed that state  isn't used now (before my change) don't know the history. Generally models in the GUI are backed by a State all the time so that their transformations can be extracted etc. This method is called from Context object that maintains the state of the current model and needs to keep it up to date. 
I'm happy to replace initSystem with any other call that keeps the State up to date with Property changes due to removing Markers and adding new ones. Old method is obviously either wrong (since it doesn't connect the model after the replace) or assumes another call before or after which is a bad practice anyway.